src/Analyzers/CSharp/Tests/ConvertNamespace/ConvertToBlockScopedNamespaceAnalyzerTests.cs

```diff
@@ -100,18 +112,51 @@ namespace N
             }.RunAsync();
         }
 
-        [Fact]
-        public async Task TestConvertToBlockSpan()
+        [Theory]
+        [MemberData(nameof(EndOfDocumentSequences))]
+        public async Task TestConvertToBlockScopedInCSharp10WithDirectives1(string endOfDocumentSequence)
         {
             await new VerifyCS.Test
             {
-                TestCode = """
+                TestCode = $$"""
                 [|namespace N;|]
+
+                #if true
+                #endif{{endOfDocumentSequence}}
                 """,
-                FixedCode = """
+                FixedCode = $$"""
                 namespace N
                 {
+                #if true
+                #endif
+                }{{endOfDocumentSequence}}
+                """,
+                LanguageVersion = LanguageVersion.CSharp10,
+                Options =
+                {
+                    { CSharpCodeStyleOptions.NamespaceDeclarations, NamespaceDeclarationPreference.BlockScoped }
                 }
+            }.RunAsync();
+        }
```

Comment:
i am not locing how these tests changed.  it has mde it harder to understand.

Reply:
➡️ This is a new test. Not sure why GitHub is saying it changed.